DataManagementSystem/Client/FTS3Operation.py

```diff
@@ -103,17 +103,26 @@ def __init__(self, ftsFiles=None, username=None, userGroup=None, rmsReqID=-1,
     self.reqClient = None
     self.dManager = None
     self._log = None
+    self.fts3Plugin = None
     self.init_on_load()
 
   @orm.reconstructor
   def init_on_load(self):
     """ This method initializes some attributes.
         It is called by sqlalchemy (which does not call __init__)
+
     """
     self._vo = None
 
+    # Note that in the case of an FTS3Operation created from an RMS
```

Comment:
The RMS operation has the usergroup passed along, and from the usergroup we get the vo (see below in the code), so this has the correct vo, doesn't it?
https://github.com/DIRACGrid/DIRAC/blob/f20575f871aae83df969d494e4cbb97651c3e70f/src/DIRAC/DataManagementSystem/Client/FTS3Operation.py#L355

Reply:
No because the `init_on_load` is called either by `SQLALchemy` or by the `__init__`. And in `fromRMSObjects` we instanciate an `FTS3Operation` object before setting the attributes, so the `init_on_load` does not have correct values yet. I can't see why I did it this way though... you want me to "fix" it ?

Reply:
Ahh, OK. (Not going to pretend I follow the logic of instantiation at this point.)
Fine for me either way.